lib/grape/middleware/formatter.rb

```diff
@@ -110,7 +110,7 @@ def read_rack_input(body)
           rescue Grape::Exceptions::Base => e
             raise e
           rescue StandardError => e
-            throw :error, status: 400, message: e.message
+            throw :error, status: 400, message: e.message, backtrace: e.backtrace, exception: e
```

Comment:
This is still `exception: ...`, which tells me there's no tests for it cause this doesn't work.